.github/workflows/node.js.yml

```diff
@@ -17,18 +17,17 @@ env:
 jobs:
   build-and-test:
     runs-on: ubuntu-latest
-    name: ${{ matrix.BROWSER }} - jQuery ${{ matrix.JQUERY }}
+    name: |
+      ${{ matrix.BROWSER }} | ${{ matrix.JQUERYS.name }}
     strategy:
       fail-fast: false
       matrix:
         BROWSER: [chrome, firefox]
-        JQUERY:
-          - "git"
-          - "3.x-git"
-          - "3.7.1"
-          - "3.6.4"
-          - "2.2.4"
-          - "1.12.4"
+        JQUERYS:
+          - versions: --jquery git --jquery 3.x-git
+            name: jQuery git
+          - versions: --jquery 3.7.1 --jquery 3.6.4 --jquery 2.2.4 --jquery 1.12.4
```

Comment:
Is it possible to make it a global variable to not repeat it 3 times or is the answer still no?

Reply:
Unfortunately, no. Not in a matrix.